docs/docs/reference/docgen/javascript/storage/content/hasura-storage-client/content/01-upload.mdx

```diff
@@ -3,16 +3,16 @@
 title: upload()
 sidebar_label: upload()
 slug: /reference/javascript/storage/upload
-description: Use `.upload` to upload a file.
-custom_edit_url: https://github.com/nhost/nhost/edit/main/packages/hasura-storage-client.ts#L32
+description: Use `nhost.storage.upload` to upload a file. The `file` must be of type `File` (https://developer.mozilla.org/en-US/docs/Web/API/File).
+custom_edit_url: https://github.com/nhost/nhost/edit/main/packages/hasura-storage-client.ts#L41
```

Comment:
Something is not right with storage edit URLs. They are not pointing to `packages/hasura-storage-js/src/hasura-storage-client.ts` file.